src/main/generic/consensus/transaction/Transaction.js

```diff
@@ -139,7 +139,7 @@ class Transaction {
     toString() {
         return `Transaction{`
             + `senderPubKey=${this._senderPubKey.toBase64()}, `
-            + `recipientAddr=${this._recipientAddr.toBase64()}, `
+            + `recipientAddr=${this._recipientAddr}, `
```

Comment:
Its actually not, because I need the recipient address in the front-end, and if I want to be able to get the HEX there, it must not be converted to base64 here.